proto/umee/ibctransfer/v1/tx.proto

```diff
@@ -0,0 +1,73 @@
+syntax = "proto3";
+package umee.ibctransfer.v1;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/duration.proto";
+import "cosmos/msg/v1/msg.proto";
+import "cosmos_proto/cosmos.proto";
+
+option go_package = "github.com/umee-network/umee/v3/x/ibctransfer";
+
+option (gogoproto.goproto_getters_all) = false;
+option (gogoproto.messagename_all)     = true;
+
+// Msg defines the x/ibctransfer module's Msg service.
+service Msg {
+  // UpdateIBCDenomsRateLimit adds new rate limits for ibc denoms or
+  // updates the rate limits for existed ibc denoms.
+  rpc UpdateIBCDenomsRateLimit(MsgUpdateIBCDenomsRateLimit) returns (MsgUpdateIBCDenomsRateLimitResponse);
+
+  // UpdateIBCTransferPauseStatus
+  rpc UpdateIBCTransferPauseStatus(MsgUpdateIBCTransferPauseStatus) returns (MsgUpdateIBCTransferPauseStatusResponse);
```

Comment:
Let's use `Gov` prefix in bot messages above, and use Quota instead of RateLimit, and reduce the length (we are in the IBC module, so no need to repeat IBC):
```
rpc GovUpdateQuota(MsgGovUpdateQuota) returns (GovMsgUpdateQuotaResponse);
```